CHANGELOG.md

```diff
@@ -22,7 +22,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `Export` class
 - `Import` class
 
+## [0.2] - 2023-08-2
+### Added
+- scikit-learn neural network models 
+- `MLP Regressor` model 
+- `MLP Classifier` model
+- `BernoulliRBN` model
+- `SGDOptimizer` transporter
+- `RandomState(MT19937)` transporter
+- `Adamoptimizer` transporter
+- Neural Network chain
+- Neural Network exceptions 
+### Changed
+- `GeneralDataStructure` Transporter updated
+- `LabelBinerizer` Transporter updated
+- `linear model` chain updated
+
 
-[Unreleased]: https://github.com/openscilab/pymilo/compare/v0.1...dev
+
+[Unreleased]: https://github.com/openscilab/pymilo/compare/v0.2...dev
 [0.1]: https://github.com/openscilab/pymilo/compare/e887108...v0.1
+[0.2]: https://github.com/openscilab/pymilo/compare/v.0.1...v0.2
+
```

Comment:
You violated the pattern. Try looking at `release` PRs in pycm.

Reply:
Try to use a better word than violation.
Ok sure, I will recheck the details.
Surly it wasn't intentional.